src/base/memory_monitor_common.h

```diff
@@ -26,54 +26,59 @@
 
 #include <cstdint>
 
-typedef struct memmon_mem_stat
+typedef enum mmon_module_id
+{
+  MMON_MODULE_LAST = 1
+} MMON_MODULE_ID;
+
+typedef struct mmon_output_stat
 {
   uint64_t init_stat;
   uint64_t cur_stat;
   uint64_t peak_stat;
   uint32_t expand_count;
-} MEMMON_MEM_STAT;
+} MMON_OUTPUT_STAT;
 
-typedef struct memmon_server_info
+typedef struct mmon_server_info
 {
   char name[DB_MAX_IDENTIFIER_LENGTH];
   uint64_t total_mem_usage;
-} MEMMON_SERVER_INFO;
+} MMON_SERVER_INFO;
 
-typedef struct memmon_subcomp_info
+typedef struct mmon_subcomp_info
 {
   char name[DB_MAX_IDENTIFIER_LENGTH];
   uint64_t cur_stat;
-} MEMMON_SUBCOMP_INFO;
+} MMON_SUBCOMP_INFO;
 
-typedef struct memmon_comp_info
+typedef struct mmon_comp_info
 {
   char name[DB_MAX_IDENTIFIER_LENGTH];
-  MEMMON_MEM_STAT stat;
+  MMON_OUTPUT_STAT stat;
   uint32_t num_subcomp;
-  MEMMON_SUBCOMP_INFO *subcomp_info;
-} MEMMON_COMP_INFO;
+  MMON_SUBCOMP_INFO *subcomp_info;
+} MMON_COMP_INFO;
```

Comment:
hmm.. _sr file also has a structure with the same name. Wouldn't it be confusing?

Reply:
I missed that during the process of applying naming rule, there could be structures with the same name. To eliminate confusion, I will rename it. How about considering the name "MMON_STAT_MAP_INFO"?

Reply:
Changed to MMON_STAT_INFO